app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -1278,6 +1273,7 @@ else if (CommCareApplication._().getAppResourceState() != CommCareApplication.ST
                 ssd.fromBundle(sessionRequest);
                 CommCareApplication._().getCurrentSessionWrapper().loadFromStateDescription(ssd);
                 this.startNextFetch();
+                return;
```

Comment:
Shouldn't be needed, since it is the last statement in a method with a void return type.